source/02-combat-prototype/actors/combat/Attack.gd

```diff
@@ -1,4 +1,4 @@
-extends CombatAction
+extends Command
 
 signal cooled
```

Comment:
I don't think this signal is really needed at the moment, probably later for animations or something, but only if followed by a `heated` signal or something like that, so that if the character tries to attack and the attack is still in cooldown a different animation can be played, communicating that the attack is still on cooldown.